packages/ods/src/components/phone-number/tests/behaviour/ods-phone-number.e2e.ts

```diff
@@ -274,6 +274,25 @@ describe('ods-phone-number behaviour', () => {
 
         expect(odsChangeSpy).toHaveReceivedEventTimes(0);
       });
+
+      it('should not do an infinite loop', async() => {
+        const dummyValue = '0123456789';
```

Comment:
maybe best to try with the expected value instead of the unformatted one, as this may not be equal on mutation
`const dummyValue = '+33123456789';`